source/deployment/sso-saml.rst

```diff
@@ -8,11 +8,15 @@ Mattermost can be configured to act as a SAML 2.0 Service Provider. The SAML Sin
 - **Centralized identity management.** Mattermost accounts can display user information from SAML, such as first and last name, email and username.
 - **Automatic account provisioning.** New Mattermost user accounts are automatically created the first time a user signs in with their SAML credentials on the Mattermost server.
 
-Mattermost officially supports Okta, OneLogin and Microsoft ADFS as the identity providers (IDPs), please see links below for more details on how to configure SAML with these providers. If you'd like, you may also try configuring SAML for a custom IDP.
+Mattermost officially supports Okta, OneLogin and Microsoft ADFS as the identity providers (IDPs), please see links below for more details on how to configure SAML with these providers. 
 
 .. toctree::
   :titlesonly:
 
   Okta SAML Configuration <sso-saml-okta>
   OneLogin SAML Configuration <sso-saml-onelogin.rst>
   Microsoft ADFS SAML Configuration <sso-saml-adfs>
+
+If you'd like, you may also try configuring SAML for a custom IDP.  For instance, customers have successfully set up Duo and PingFederate as a custom IDP. We are happy to provide assistance when configuring your custom IDP by answering Mattermost technical configuration questions and working with your IDP provider in support of resolving issues. Please see more information on getting support `here <https://mattermost.com/support/>`_.  
```

Comment:
> Duo and PingFederate as a custom IDP

SimpleSAMLphp was also configured successfully

> We are happy to provide assistance when configuring your custom IDP by answering Mattermost technical configuration questions and working with your IDP provider in support of resolving issues

The language makes it sound like we'll work with them until the issues are resolved. Not sure if there is a standard / expectation of whether we'd resolve all issues, and the cadence of it.

Reply:
Agreed, we can be a bit more explicit here as some issues will be out of our ability to resolve directly.  We could add "as they relate to Mattermost SAML configuration settings." Open to other suggestions as well.

Reply:
How about.

```
We are open to providing assistance when configuring your custom IDP by answering Mattermost technical configuration questions and working with your IDP provider in support of resolving issues as they relate to Mattermost SAML configuration settings. However, we cannot guarantee your connection will work with Mattermost.

Please see more information on getting support `here <https://mattermost.com/support/>`_ and submit requests for official support of a particular provider on our `feature idea forum <https://mattermost.uservoice.com>`_.
```

Reply:
Thanks, Jason -this proposal makes sense to me